tonypedraza/alloy
# Fix webpack

This PR doesn't really do anything. I just added the watch flag to webpack.

To test:
Run `npm run build` in the terminal. Not `docker run alloy_django npm run build`

Webpack should work whenever you change the p tag in the App.js file.
